packages/modules/fsm/src/components/Username.js

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+const Username = ({ assigner }) => {
+  const { t } = useTranslation();
+  const [designation, setDesignation] = useState(null);
+  const tenantId = Digit.ULBService.getCurrentTenantId();
```

Comment:
there can be issues here to call employee search from citizen state level tenant id. since employee roles are tenant specific

Reply:
I have disabled employee search for userType citizen
![Screenshot from 2021-03-27 16-55-41](https://user-images.githubusercontent.com/75366065/112719143-53578400-8f1d-11eb-92e3-f7f90736b0ae.png)